plugins/example_collector/test/test_config.py

```diff
@@ -0,0 +1,11 @@
+from resotolib.config import Config
+from resoto_plugin_example_collector import ExampleCollectorPlugin
+
+
+def test_config():
+    config = Config("dummy", "dummy")
+    ExampleCollectorPlugin.add_config(config)
+    Config.init_default_config()
+
+
+#    assert Config.example.region is None
```

Comment:
left over?

Reply:
No, just an example of how to access the Config without actually adding any examples to the user's config.